src/sentry/issues/related/__init__.py

```diff
@@ -13,9 +13,10 @@
 }
 
 
-def find_related_issues(group: Group) -> list[dict[str, list[int] | str]]:
-    related_issues: list[dict[str, list[int] | str]] = []
+def find_related_issues(group: Group) -> list[dict[str, str | list[int] | dict[str, str]]]:
+    related_issues: list[dict[str, str | list[int] | dict[str, str]]] = []
```

Comment:
Would we be able to have this as a `list[TypedDict]`, or are the fields unknown? Might make it easier to understand at a glance

Reply:
I misread the code this can be ignored